src/browser/tropy.js

```diff
@@ -394,6 +394,9 @@ class Tropy extends EventEmitter {
       this.dispatch(act.item.destroy(target.id))
     })
 
+    this.on('app:order-item', (win, { target }) =>
```

Comment:
I don't think this is used? Please remove it.

Reply:
done